### DISCUSSION TITLE
layout entry blueprint yaml field

### DISCUSSION THREAD

**Discussion by breizhwave** [statamic/cms#5795] [Q&A]
Hi, I like the possibility of customizing the layout by collection or entry via the layout yaml variable, I particularly love the way you can add a yaml field in collection to setup the layout per entry. Is there a possibility of making a select yaml field ? instead of the existing  input yaml field ?

**Comment by edalzell** [comment]
Not quite sure what you mean, can you explain?

**Comment by breizhwave** [comment]
In my "page" blue print, I have added a YAML field named "layout" : each page can have its own layout. works like a charm. But this field is INPUT, was just wondering how to make  a SELECT YAML FIELD

**Reply by jasonvarga** [reply]
I still don't really understand what you're asking, sorry. Have you tried creating a "Select" field?

**Reply by breizhwave** [reply]
Sorry you're right !! my misunderstaing, a simple select field does the job too, of course everything is YAML.

was just wondering what's the point with YAMl field , found my answer https://statamic.dev/fieldtypes/yaml

thanks

**Reply by jasonvarga** [reply]
Ah I see. Yes the `yaml` fieldtype is not a very common one to use.
